packages/airview-compliance-ui/src/features/application-tile/application-tile-title.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Typography from "@mui/material/Typography";
```

Comment:
For consistency, import as a named import: 

`import { Typography } from "@mui/material";`